include/WCSimPrimaryGeneratorMessenger.hh

```diff
@@ -38,6 +38,8 @@ class WCSimPrimaryGeneratorMessenger: public G4UImessenger
   G4UIcmdWithABool*   poisCmd;
   G4UIcmdWithADouble* poisMeanCmd;
   G4UIcmdWithAString* fileNameCmdCosmics;
+  G4UIcmdWithAnInteger* seedNumberCmdAmBe;
+  G4UIcmdWithABool*   BGOEvent;
```

Comment:
```suggestion
```
These seem to not be used. Please delete here (and where it's used in the .cc)

Reply:
This appears to not have been done yet

Reply:
👍